### ISSUE TITLE
GUI Daemon / Agent global clipboard mechanism is not INCR aware

### ISSUE THREAD

**Issue by alimirjamali** [QubesOS/qubes-issues#9494]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release
r4.3 but most probably way back to r3.2 (or maybe even older).

### Brief summary
If you try to copy text over 256KiB to Global Clipboard, just one useless byte is copied.

### Steps to reproduce
- Create a text file over 256KiB. Maybe using output of this simple Python script:
```python
#!/bin/python
a = "x" * 511 + "\n"
b = a * 1024
```
- Open it in a text editor (e.g. mousepad).
- Select all text and press `ctrl+c`
- Copy to global clipboard with `ctrl+shift+c`
- Observe that Global Clipboard informs you that 1 byte is copied to global clipboard

### Expected behavior
Either reject inter-vm INCR clipboard or support it after #9296 is resolved via the purposed patch.

### Actual behavior
Confusing behaviour with clipboard sizes over 256KiB

**Comment by alimirjamali** [comment]
This is essentially duplicate of #7771 & #5220. But those two issues lack the technical details about the issue. Overall it is easy to fix after the fix for #9296 is decided. I am going to close this for the time.
